runtime/loader/src/main/java/com/walmartlabs/concord/process/loader/v2/ConfigurationV2.java

```diff
@@ -27,22 +27,31 @@
 import com.walmartlabs.concord.runtime.v2.model.ProcessConfiguration;
 
 import java.io.Serializable;
+import java.util.List;
 import java.util.Map;
 
 public class ConfigurationV2 implements Configuration, Serializable {
 
     private static final long serialVersionUID = 1L;
 
+    private final ProcessConfiguration cfg;
     private final Map<String, Object> values;
 
     @SuppressWarnings("unchecked")
     public ConfigurationV2(ProcessConfiguration cfg) {
+        this.cfg = cfg;
+
         ObjectMapper om = new ObjectMapper();
         om.registerModule(new Jdk8Module());
         om.registerModule(new JavaTimeModule());
         this.values = om.convertValue(cfg, Map.class);
     }
 
+    @Override
+    public List<String> dependencies() {
+        return cfg.dependencies();
```

Comment:
Why not to use the `values` object like in `ConfigurationV1`?
This way we can avoid saving a reference to `ProcessConfiguration` and duplicated data if someone serializes a `ConfigurationV2` instance.

Reply:
fixed